src/components/gamepage/GamesPageImageGrid/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+// line 7-8-9 are mock datas
+import zelda from "../../../images/mockZeldaPoster.jpg";
+import gta from "../../../images/mockGTAPoster.jpg";
+import destiny from "../../../images/mockDestinyPoster.jpg";
+
+const mockDataArr = [
```

Comment:
MOCK_DATA_ARR (this type of casing), and give it a better name. data for what?

Grand Theft Auto

also please make sure this is fixed by demo day

Reply:
Name fixed. I will fix everything once all games are finished.

Reply:
PTAL